Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm

```diff
@@ -233,6 +239,36 @@ - (TextMarkerData)textMarkerData
 
 @implementation WebAccessibilityObjectWrapper
 
+
+#if ENABLE(SPATIAL_IMAGE_CONTROLS)
+- (NSArray *)_accessibilitySupplementaryFooterViews
+{
+    if (![self _prepareAccessibilityCall])
+        return nil;
+
+    auto* backingObject = self.axBackingObject;
```

Comment:
I think this should be a RefPtr to ensure the object stays alive for the duration of the function.